src/styles/index.css

```diff
@@ -0,0 +1,12 @@
+body {
+  margin: 0;
+  font-family: "Nunito", sans-serif;
```

Comment:
Is nunito being used anywhere? It seems like Poppins is the font of choice, that could be imported here as the main font which would mean you dont have to keep imported it into all the other files as it would cascade down